rviz_common/include/rviz_common/ros_topic_display.hpp

```diff
@@ -98,6 +98,9 @@ protected Q_SLOTS:
   virtual void transformerChangedCallback()
   {
   }
+  virtual void updateMessageQueueSize()
```

Comment:
Is adding this virtual method necessary? I guess it's sufficient to only add it to the child class (which has the concept of queue size).

Reply:
I believe so. If I remove this virtual method, the slot method doesn't seem to link correctly. The warning is below. I think it has something to do with the `Q_SLOTS` macro above the virtual method.

```
QObject::connect: No such slot rviz_common::_RosTopicDisplay::updateMessageQueueSize()
QObject::connect:  (sender name:   'Filter size')
```

Reply:
Ah, yeah that's right. I guess we can add Q_SLOTS to the child class because it's a template class.